app/components/button/button.tsx

```diff
@@ -9,14 +9,16 @@ const Button: FC<{
   outline?: boolean
   main?: boolean
   style?: Record<string, string | number>
-}> = ({ children, outline, main, style = {} }) => {
+  onClick?: MouseEventHandler<HTMLDivElement> | undefined
+}> = ({ children, outline, main, style = {}, onClick }) => {
   return (
     <div
       className={clsx(s.button, {
         [s['button-outline']]: outline,
```

Comment:
difficult naming for .scss

Reply:
common bro, it's just name of a class, get them kill itself as not the best practice which no one use it =))